.moai/reports/v311-docs/docs-4locale-surface-map.md

```
@@ -0,0 +1,83 @@
+| guides | 4 | yes |
+| worktree | 4 | yes |
+| advanced | 32 | yes |
+| contributing | 1 | yes |
+| changelog | 1 | no |
+| claude-code (+4 subdirs: foundations 9, context-memory 5, extensibility 5, agentic 10) | 30 | no |
+
+Section weights identical across locales: getting-started 10, resources 15, core-concepts 20,
+workflow-commands 30, utility-commands 40, cli-reference 45, multi-llm 60, cost-optimization 70,
+guides 85, worktree 90, advanced 100, contributing 110, claude-code 130.
+
+### Menu gaps (pages exist, no `main.yaml` entry)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)